src/main/java/com/faforever/client/chat/PrivateChatTabController.java

```diff
@@ -51,9 +98,118 @@ public void setReceiver(String username) {
     super.setReceiver(username);
     privateChatTabRoot.setId(username);
     privateChatTabRoot.setText(username);
+
+    //Load receiver information for detail pane
+    userPlayer = playerService.getPlayerForUsername(username);
+    if (userPlayer != null) {
```

Comment:
extracted to own method, `if (userPlayer != null) {
      loadDetailPane(userPlayer);
    }`